geps/gep-1619.md

```diff
@@ -210,18 +666,157 @@ TBD - See [open question](#open-questions) about session affinity API
 - Should we include both session persistence and session affinity API configurations in this GEP or just focus on session persistence?
```

Comment:
My vote is to only focus on session persistence here and move session affinity API to a different GEP

Reply:
I was starting to feel that they have diverged. I believe we came to a tentative agreement that we are keeping the APIs separated. I'd think doing the clean up in a separate PR would be better to simplify this one.

I'm all for separating to a different GEP, but the only impact that SessionAffinity could have on the outcome of this SessionPersistence GEP, is if we went with our [LoadBalancerPolicy](https://github.com/kubernetes-sigs/gateway-api/pull/2159/files/a377c3d7cf31ffc6c1daf1acea8275e9c39ac64f#diff-afa0e827962fe48152635217c6bc954020d5fec3578af4a9e5f744a2b4568fffR767-R774) alternative.

The alternative would build a common API to house both Session Persistence and Session Affinity. I just want to be sure we don't want to pursue that design, and instead, keep things granular like we are in the current design. I feel like we have pushed other PolicyAttachment-type GEPs, like the [Backend TLS GEP](https://github.com/kubernetes-sigs/gateway-api/pull/2113) (previously BackendCapabilities) to being more and more granular. Thoughts?

Reply:
Agree with keeping it more granular, instead of going with the [LoadBalancerPolicy](https://github.com/kubernetes-sigs/gateway-api/pull/2159/files/a377c3d7cf31ffc6c1daf1acea8275e9c39ac64f#diff-afa0e827962fe48152635217c6bc954020d5fec3578af4a9e5f744a2b4568fffR767-R774) alternative